extension-spring/src/test/java/io/smallrye/openapi/runtime/scanner/SpringAnnotationScannerTest.java

```diff
@@ -33,4 +33,23 @@ public void testBasicSpringDefinitionScanning() throws IOException, JSONExceptio
         assertJsonEquals("resource.testBasicSpringDefinitionScanning.json", result);
     }
 
+    @Test
+    public void testUserControllerScanning() throws IOException, JSONException {
+        Indexer indexer = new Indexer();
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/resources/PetController.class");
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/resources/UserController.class");
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/resources/StoreController.class");
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/entities/Category.class");
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/entities/Order.class");
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/entities/Pet.class");
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/entities/Tag.class");
+        index(indexer, "test/io/smallrye/openapi/runtime/scanner/entities/User.class");
+        OpenApiAnnotationScanner scanner = new OpenApiAnnotationScanner(emptyConfig(), indexer.complete());
+
+        OpenAPI result = scanner.scan();
+
+        printToConsole(result);
+        assertJsonEquals("resource.testBasicSpringDefinitionScanning.json", result);
```

Comment:
Is there a different test JSON file that should be used here?

Reply:
I didn't bother to make one yet. ;)
I said this was a draft. What was being produced is still unique to smallrye (in terms of which default strings are used). I will make a json result when what we're producing is close enough that I can define what the correct result is.